deploy/docker-compose/README.md

```diff
@@ -0,0 +1,18 @@
+# Run SuperDuperDB using Docker-Compose
+
+
+#### Build and run your app with Compose
+```shell
+ docker-compose -f demo.yaml up 
+```
+
+Have in mind that `docker-compose` with source the `default credentials` from `.env` file.
+
+#### Access Jupyter
+Jupyter's access mechanism is based on the concept of authentication tokens. 
+These tokens are normally written in the logs of `docker-compose`, but some times it can get messy.
+
+To get the token directly, use:
+```shell
+docker logs docker-compose-superduperdb-1 |& grep -m 2 "/lab?token="
```

Comment:
Should we redirect the token to stdout of docker?

Reply:
We should also print the whole URL so that users can just cut and paste into a browser.

Reply:
It is printed in the stdout of docker-compose. But since the stdout if used by both mongo and jupyter, the token is likely to be lost in a sea of logs.

Reply:
I went through by disabling the authentication tokens.